src/queries/waterTemperature.js

```diff
@@ -0,0 +1,12 @@
+import { waterTemperatureEndpoint } from '../config';
+
+export const getWaterTemperature = async () => {
+  const { serverUrl, authSecret } = waterTemperatureEndpoint;
+
+  const fetchObj = {
+    method: 'GET',
+    headers: { Authorization: `readonly:${authSecret}` }
```

Comment:
i will add the whole `Authorization` as `authSecret`. There is no good reason to keep the `readonly` here and the code there.